examples/pymatgen.jl

```diff
@@ -0,0 +1,37 @@
+# # Creating supercells with pymatgen
+#
+# The [Pymatgen](https://pymatgen.org/) python library allows to setup
+# solid-state calculations using a flexible set of classes as well as an API
+# to an online data base of structures. Its `Structure` and `Lattice`
+# objects are directly supported by the DFTK `load_atoms` and `load_lattice`
+# functions, such that DFTK may be readily used to run calculation on systems
+# defined in pymatgen. Using the `pymatgen_structure` function a conversion
+# from DFTK to pymatgen structures is also possible. In the following we
+# use this to create a silicon supercell and find its LDA ground state
+# using direct minimisation.
```

Comment:
"direct minimization (SCF is boring)" or something - just to say that we could just as well do an SCF